.github/workflows/end2end-test.yml

```
@@ -125,8 +126,9 @@ jobs:
               # Don't fail the job if there isn't any flaky tests report.
               continue-on-error: true
               with:
-                  name: flaky-tests-report
+                  name: flaky-tests-report-*
```

I think it should be `pattern` here 😅
```suggestion
                  pattern: flaky-tests-report-*
```

---

I reviewed this 2 weeks ago but I never submit this 🤦

---

🤦‍♂️ on my end too! Should be fixed now!